packages/dx-react-scheduler-material-ui/src/templates/drag-drop/appointments.jsx

```diff
@@ -95,10 +108,12 @@ AppointmentBase.propTypes = {
   durationType: PropTypes.string,
   className: PropTypes.string,
   type: PropTypes.string,
+  isShadedAppointment: PropTypes.bool,
 };
 
 AppointmentBase.defaultProps = {
   durationType: undefined,
   className: undefined,
   type: undefined,
+  isShadedAppointment: false,
```

Comment:
Can we use only `isShaded` instead of `isShadedAppointment`, like you are doing in the Cell component. I think the word `appointment` is excess in this context.

Also in Appointment component